crates/router/src/connector/adyen.rs

```diff
@@ -1910,3 +1910,324 @@ impl api::IncomingWebhook for Adyen {
         })
     }
 }
+
+impl api::Dispute for Adyen {}
+impl api::DefendDispute for Adyen {}
+impl api::AcceptDispute for Adyen {}
+impl api::SubmitEvidence for Adyen {}
+
+impl
+    services::ConnectorIntegration<
+        api::Accept,
+        types::AcceptDisputeRequestData,
+        types::AcceptDisputeResponse,
+    > for Adyen
+{
+    fn get_headers(
+        &self,
+        req: &types::AcceptDisputeRouterData,
+        _connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, request::Maskable<String>)>, errors::ConnectorError> {
+        let mut header = vec![(
+            headers::CONTENT_TYPE.to_string(),
+            types::AcceptDisputeType::get_content_type(self)
+                .to_string()
+                .into(),
+        )];
+        let mut api_key = self.get_auth_header(&req.connector_auth_type)?;
+        header.append(&mut api_key);
+        Ok(header)
+    }
+
+    fn get_url(
+        &self,
+        req: &types::AcceptDisputeRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<String, errors::ConnectorError> {
+        let endpoint = build_env_specific_endpoint(
+            connectors.adyen.third_base_url.as_str(),
+            req.test_mode,
+            &req.connector_meta_data,
+        )?;
+        Ok(format!("{}acceptDispute", endpoint))
+    }
+
+    fn build_request(
+        &self,
+        req: &types::AcceptDisputeRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<Option<services::Request>, errors::ConnectorError> {
+        Ok(Some(
+            services::RequestBuilder::new()
+                .method(services::Method::Post)
+                .url(&types::AcceptDisputeType::get_url(self, req, connectors)?)
+                .attach_default_headers()
+                .headers(types::AcceptDisputeType::get_headers(
+                    self, req, connectors,
+                )?)
+                .set_body(types::AcceptDisputeType::get_request_body(
+                    self, req, connectors,
+                )?)
+                .build(),
+        ))
+    }
+    fn get_request_body(
+        &self,
+        req: &types::AcceptDisputeRouterData,
+        _connectors: &settings::Connectors,
+    ) -> CustomResult<RequestContent, errors::ConnectorError> {
+        let merchant_account_code = get_key(&req.connector_auth_type)?;
+        let connector_req =
+            adyen::AdyenAcceptDisputeRequest::try_from((req, merchant_account_code))?;
+        Ok(RequestContent::Json(Box::new(connector_req)))
+    }
+
+    fn handle_response(
+        &self,
+        data: &types::AcceptDisputeRouterData,
+        _event_builder: Option<&mut ConnectorEvent>,
+        _res: types::Response,
+    ) -> CustomResult<types::AcceptDisputeRouterData, errors::ConnectorError> {
+        Ok(types::AcceptDisputeRouterData {
+            response: Ok(types::AcceptDisputeResponse {
+                dispute_status: api::enums::DisputeStatus::DisputeAccepted,
```

Comment:
We can't hardcode the status here, when we get a 200 response.

Instead, we need to consume the body and decide the status
```
{
  "disputeServiceResult": {
    "success": true
  }
}
```